pym/bob/input.py

```diff
@@ -108,10 +108,12 @@ def sliceHost(digest):
         """Extract host fingerprint digest part (if any)."""
         return digest[20:]
 
-def fetchScripts(recipe, prefix, resolveBash = lambda x, y: x):
+def fetchScripts(recipe, prefix, resolveBash = lambda x, y: x, resolvePwsh = lambda x, y: x):
     return {
         ScriptLanguage.BASH : resolveBash(recipe.get(prefix + "ScriptBash",
             recipe.get(prefix + "Script")), prefix + "Script[Bash]"),
+        ScriptLanguage.PWSH : resolvePwsh(recipe.get(prefix + "ScriptPwsh"),
+            prefix + "ScriptPwsh"),
```

Comment:
Hi,
I suppose that's what the expression should be: `ScriptLanguage.PWSH : resolvePwsh(recipe.get(prefix + "ScriptPwsh", recipe.get(prefix + "Script")), prefix + "Script[Pwsh]")` ? Without this only the package step is executed because no scripts for checkout and build step can't be fetched. I tested the change in my environment (native windows) and everything is building fine again.

Reply:
Actually not. The idea was that `*Script` is just an alias for `*ScriptBash`. If you want to use PowerShell scripts they always have to be written in `*SriptPwsh`.

*BUT*, I think your idea makes sense. Then the `*Script` will be taken for whatever language is used to build the recipe. Classes can use `*ScriptBash` and `*scriptPwsh` if they like to support both. I will just have to update the documentation...